shared/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -925,10 +925,10 @@ Undocumented.prototype.updateCreditCard = function( params, fn ) {
  * @param {Function} fn The callback function
  * @api public
  */
-Undocumented.prototype.paygateConfiguration = function( fn ) {
+Undocumented.prototype.paygateConfiguration = function( data, fn ) {
 	debug( '/me/paygate-configuration query' );
 
-	this.wpcom.req.get( '/me/paygate-configuration', fn );
+	this.wpcom.req.post( '/me/paygate-configuration', data, fn );
```

Comment:
We always read existing data from the server. We don't create or update anything in this case. Therefore we should keep using `GET` REST API method.